exercises/linked-list/linked_list_test.go

```diff
@@ -1,3 +1,19 @@
+// API:
+//
+// type Element struct
+// type List struct 
```

Comment:
One extra space at the end of the line caused 'go fmt' to make a change, and it failed the travis-ci.